tests/testthat/test-cache.R

```diff
@@ -92,10 +92,19 @@ test_that("load_cache returns an empty environment if reading cache file fails",
   cache_f1 <- file.path(d1, fhash(f1))
   writeLines(character(), cache_f1)
 
-  expect_warning(e2 <- lintr:::load_cache(file = f1, path = d1), "Could not load cache file")
+  expect_warning(
+    {
```

Comment:
I found myself ready to suggest nesting the expectations instead & then remembered that's basically what we just undid by removing `magrittr` :)

Someday we'll depend on R>4.1 & then we can use `|>`....

In the meantime, there may be some cases where 2-nesting looks fine:

```r
expect_warning(
  expect_identical(ls(lintr:::load_cache(file = f1, path = d1), character()),
  "Could not load cache file",
  fixed = TRUE
)
```

up to you whether / which cases you agree

Reply:
TBH, I personally prefer to keep each expectation separate. Nested expectations are much harder to reason about.